packages/zoe/tools/types.js

```diff
@@ -14,7 +14,7 @@
  * @typedef {Object} PriceDescription A description of a single quote
  * @property {Amount} amountIn The amount supplied to a trade
  * @property {Amount} amountOut The quoted result of trading `amountIn`
- * @property {TimerService} timer The service that gave the `timestamp`
+ * @property {ERef<TimerService>} timer The service that gave the `timestamp`
```

Comment:
I suggest reverting this.  The PriceDescription needs to be comparable, and that means the `timer` should be `await`ed before putting it in the description.